integrations/tensorflow/e2e/BUILD

```diff
@@ -93,6 +93,7 @@ VULKAN_FAILING = [
     "matrix_ops_test.py",
     "ring_buffer_test.py",  # TODO(b/148747011)
     "scatter_update_test.py",
+    "sliding_window_test.py", # TODO(#2659)
```

Comment:
buildifier is upset about the lack of space here :-D

Please run buildifier (no need to rerun tests after though)

Reply:
Need to burn more CPU cycles for one space. ;-P